scrunch/tests/test_datasets.py

```diff
@@ -3591,3 +3592,47 @@ def test_filter_class(self):
         filter = MagicMock(entity=self._filter)
         mockfilter = Filter(filter)
         assert mockfilter
+
+
+class TestDeck(TestDatasetBase, TestCase):
```

Comment:
Should test your `.decks` property accessor, it has a bug mentioned above when trying to fetch it by name.